### DISCUSSION TITLE
QInput and Qselecе jeight

### DISCUSSION THREAD

**Discussion by albukzek** [quasarframework/quasar#13772] [General - Components / Directives / etc]
Hello.
How to change the height of an element in quasar?
According to the layout, I have a select height of 36 pixels, but if you set such a height, the select layout collapses. The same with imput.

How to make a low imput and select using a quasar?

Sorry for my English. Used a translator :)

**Comment by MilosPaunovic** [comment]
Either use `dense` prop or by CSS.

**Reply by albukzek** [reply]
> Either use `dense` prop or by CSS.

Thanks for the answer.
Dense is not enough. And in CSS, the styles will change the size, but very badly. See example
![123](https://user-images.githubusercontent.com/31003788/175521838-cac37490-8376-467b-9ee4-101cfc891d62.PNG)

the light rectangle at the top inside is the CSS, and the big contour is what came out of the quasar

**Comment by MilosPaunovic** [comment]
Please provide a Codepen for someone to be able to take a deeper dive.

**Reply by albukzek** [reply]
> Please provide a Codepen for someone to be able to take a deeper dive.

https://codepen.io/albukzek/pen/PoQMKWd
I don’t know how the quasar will connect there, but I wrote part of the input code

**Reply by MilosPaunovic** [reply]
Use this as a base:
[https://codepen.io/milospaunovic/pen/MWQNvvw?editors=101](https://codepen.io/milospaunovic/pen/MWQNvvw?editors=101)

**Reply by albukzek** [reply]
> Use this as a base: https://codepen.io/milospaunovic/pen/MWQNvvw?editors=101

Thanks, updated.
Gray there is what size the input should be. But Quasar thinks otherwise :)

https://codepen.io/albukzek/pen/PoQMKWd

**Comment by MilosPaunovic** [comment]
[https://codepen.io/milospaunovic/pen/GRQVMOR](https://codepen.io/milospaunovic/pen/GRQVMOR)
